code/modules/mob/language.dm

```diff
@@ -406,6 +406,15 @@
 	key = "3"
 	syllables = list ("gra","ba","ba","breh","bra","rah","dur","ra","ro","gro","go","ber","bar","geh","heh", "gra")
 
+/datum/language/sign
+	name = "Sign Language"
+	desc = "A non-verbal form of communication that utilizes hand gestures to form words and meanings."
+	speech_verb = "signs"
+	signlang_verb = list("signs", "gestures")
+	colour = "say_quote"
+	key = "~"
```

Comment:
I have tried to use ~ as a key before, and if memory serves, it does not work.

Reply:
Oh, I was sure that it worked when I tested it on a local server instance, but I'm happy to change this to a different character. Any recommendations, all the good characters seem to be in use already :/

I did originally try to use :si , as other languages seemed to have two character codes, but it just outputted to the security channel.

Reply:
Frankly, almost all of the keys are taken, and double-letter keys don't work.

You might be better off making it a 'spell' (ie: giving it its own HUD icon). Not sure if this will be popular, but it is one method to get around the key requirement.

Alternatively, you could find an existing key that is never used, remove it, and re-purpose its key for this.

Reply:
Further testing on my local server seems to suggest ~ is working for me.

If i have the time, I might look into seeing if I can fix double-letter keys not working, that would open up a lot more combinations for future use.